### DISCUSSION TITLE
Netradiant Customs self creating folders.

### DISCUSSION THREAD

**Discussion by Hipshot-zz** [func-godot/func_godot_plugin#12] [General]
Ok, so I have been talking a bit to the dev of NRC about this, to a level where I feel that he might be a bit annoyed by me haha and I feel that this is maybe not really his fault, but maybe old NetRadiants fault. But:

It seems to me that NRC is forcingly creating a folder structure you might not want in your project and as far as I know, there's no way of stopping it from doing that. Unless someone here knows how.

It seems NRC is ALWAYS creating an "/asset" folder and AT LEAST" a "/maps" folder in your project. Even if you want to have lets say the maps in uh, "world" or "levels" or whatever. The asset folder is alright, because that can be just your roots name but you can't get rid of the /maps part, which is terrible for a project outside a "quake-game" I feel.

Maybe this could be mentioned in the documentation at least, because I have now moved my .map into a /maps folder, even if I really didn't want to, just because I couldn't get rid of the folder (feels stupid to have an empty folder you know). You can't delete it, because it's recreated every time you load NRC. I suppose I can just make a script that deletes the folder, but you know... nah =)

If anyone have any idea here, please fill me in, maybe I'm doing something wrong.

**Comment by sinewavey** [comment]
+1. 

For me this takes the firm of a base game path, say `sdk`. It'll auto populate `scripts` and `maps`. 

I just ended up storing my maps where it was laid out. 

Just checking, as currently I'm away from my computer- are settings for this in the game pack file, perhaps?

**Comment by Hipshot-zz** [comment]
You can get rid of scripts there yes, but not maps.
Scripts are created via this: shaderpath="scripts" just change that to "" and the scripts folder won't spawn, but there's no such things for maps.

I think this is a very annoying behaviour, the editor shouldn't create any folder as I see it. I have never experienced something that forces this, especially an "old" editor like this. I just wish there could be a way for this to be togglable at least, somewhere, could even be buried very deep.

**Reply by sinewavey** [reply]
I do use the scripts for shaders in NRC myself but I agree it's annoying to force generate anything you don't say outright to

**Comment by Hipshot-zz** [comment]
Yea, I just think it could be written in the documentation section of NRC for func_godot.
